### PR TITLE
24 adding SSL with updated info

### PR DESCRIPTION
**Pull request by BlackNinjaCat86**
I have updated the SSL protocol with new info.

### CODE REVIEW

File: content/topics/protocols/index.md [hunk 1]
```diff
@@ -179,3 +179,15 @@ In corporate networks, SSH protocol can be used to guarantee secure access for u
 **References:**
 
 * [SSH-SSH Academy](https://www.ssh.com/academy/ssh/protocol)
+
+### SSL **Secure Sockets Layer**
+
+{{< youtube id="ZghMPWGXexs&ab" title="SSL Encryption and decrytion" >}}
```
Comment (schalkneethling) [inline content/topics/protocols/index.md:185]: As mentioned on another pull request, perhaps reword this to something like:

```suggestion
{{< youtube id="ZghMPWGXexs&ab" title="A video explaining SSL encryption and decryption" >}}
```

File: content/topics/protocols/index.md [hunk 2]
```diff
@@ -179,3 +179,15 @@ In corporate networks, SSH protocol can be used to guarantee secure access for u
...
+
+This is a networking protocol that keeps your internet connection secure and ensures that data being transfered remains private. This protocols' main use is securing the connection/communication between your browser and a web server. It does this by scrambling the data and making it impossible to read to anyone trying to intercept it. SSL uses cryptographic keys to encrypt information; a public key encrypts the data and a private key decrypts it (making it readable). An SSL certificate enables websites to move from HTTP to HTTPS.
```
Comment (schalkneethling) [inline content/topics/protocols/index.md:187]:
```suggestion
Secure socket layer (SSL) is a networking protocol that keeps your internet connection secure and ensures that the data being transferred remains private. This protocol's main use is securing the connection/communication between your browser and a web server. It does this by scrambling the data and making it impossible to read to anyone trying to intercept it. SSL uses cryptographic keys to encrypt information; a public key encrypts the data, and a private key decrypts it (making it readable). An SSL certificate enables websites to move from HTTP to HTTPS.
```

Seeing that we are mentioning SSL certificates, it would be helpful to dig into it a little bit more. https://www.cloudflare.com/learning/ssl/what-is-an-ssl-certificate/

This is a useful browser extension that would be good to mention. https://www.eff.org/https-everywhere

Also, SSL certificates used to be incredibly expensive and hard to come by. A non-profit called Let's Encrypt changed all of this and is a large part of why most, if not all, of the internet is now served over HTTPS. I would be good to also mention them and a little bit of the history behind it all. https://letsencrypt.org/about/

**Review by schalkneethling — CHANGES_REQUESTED** [review]
Thank you for digging into this some more, @BlackNinjaCat86 :+1: I provided some more context and history we should also dig into. Thanks!

**Comment by BlackNinjaCat86** [comment]
@schalkneethling, all changes have been made 😄

**Review by schalkneethling — APPROVED** [review]
Thank you, @BlackNinjaCat86 🌮
